.github/workflows/ci.yml

```diff
@@ -45,9 +45,9 @@ jobs:
     strategy:
       matrix:
         images:
-          - githubRegistry: ghcr.io/ketch-com/ketch-event-forwarder
+          - localRegistry: localhost:5000/ketch-com/ketch-event-forwarder
+            # githubRegistry: ghcr.io/ketch-com/ketch-event-forwarder
             # ecrRepository: ketch-event-forwarder
```

Comment:
These two variables are used in the Docker meta step. You need to clean that up.